packages/xo-web/src/common/xo/migrate-vm-modal/index.js

```diff
@@ -119,10 +121,6 @@ export default class MigrateVmModalBody extends BaseComponent {
     )
   }
 
-  componentDidMount () {
-    this._selectHost(this.props.host)
-  }
```

Comment:
@pdonias Is it still relevant?

Reply:
After discussion, it appears it still relevant, @Enishowk I agree with your changes.